serverless-functions/profile_retrieval/serverless.yml

```diff
@@ -76,9 +79,14 @@ functions:
     handler: handler.handle
     description: serverless-wsgi based endpoint to retrieve profiles aka person-api.
     memorySize: 3000
-    timeout: 30
+    # 15 minutes. Though, as we found out, anything above 30 seconds will get
+    # cut off by an internal timeout from the API Gateway.
+    # This value is to preserve some manual changes we've had to make along the
+    # way.
+    timeout: 900
+    logRetentionInDays: 30
     events:
       - http: ANY /
       - http: ANY {proxy+}
     layers:
-      -  ${ssm:/iam/cis/${self:custom.profileRetrievalStage}/lambda_layer_arn}
+      -  ${ssm:/iam/cis/${self:custom.profileRetrievalStage}/build/lambda_layer_arn}
```

Comment:
nitpick: this should include somewhere the architecture and runtime, probably, since we may want to publish multiple lambda layers for different combinations.

Some reasons:

* we may want to test with the same python version but different processor (arm64 should give us some savings);
* we may want to test with a different python but the same architecture (e.g. python upgrades, as we'll soon be forced to do).

I'm not going to address this now, since we're not close to this being an issue. But, it _could_ be an issue.